genref/README.md

```diff
@@ -0,0 +1,74 @@
+# API Reference generator
```

Comment:
What do you think about creating a more specific name for this tool?
Thinking about: Type [API] generator?

Reply:
How about "Config API generator"?

Almost all packages we are dealing with today are about the component config or tool config.

Reply:
OK. I need to review the generated output.

Reply:
The generated output is in a separate follow-up PR.